tools/run-tests.py

```diff
@@ -195,6 +197,45 @@ def run_jsc_stress(engine, arch):
         env={'PYTHONPATH': '.'})
 
 
+def _run_regression_tests(files, is_fail):
+    for file in files:
+        passed = True
+        msg = ''
+        code = 0
+        try:
+            out = subprocess.check_output([DEFAULT_ESCARGOT, file])
+        except subprocess.CalledProcessError as exc:
```

Comment:
Use `Popen` and `communicate` instead of `check_output` and `CalledProcessError`, examples can be found in `run` (line 60). (And no need to import `subprocess` as `Popen and `communicate` have already been imported from it.)